src/pages/AboutPage.tsx

```diff
@@ -1,8 +1,78 @@
-import React from "react";
+import React, { useState } from "react";
+import { Button } from "@material-ui/core";
 import { Header } from "components/Header";
+import { labels } from "labels";
+
+interface AboutLinkProps {
+  url: string;
+  label: string;
+}
+
+const AboutLink: React.FC<AboutLinkProps> = ({ url, label }) => {
+  return (
+    <div>
+      <a
+        style={{ textDecoration: "none" }}
+        target="_blank"
+        rel="noopener noreferrer"
+        href={url}
+      >
+        {label}
+      </a>
+    </div>
+  );
+};
 
 export const AboutPage: React.FC = () => {
+  const [aboutData] = useState({
+    server: "openrainbow.com",
+    telephonyVersion: "unknown",
+    systemIdentifier: "PBX7467-...",
+  });
+
   return (
-    <Header title="Rainbow Click2Cal" subtitle="1.7.20" toPage="settings" />
+    <div>
```

Comment:
MUI's `Box` is a better wrapper with more options for styling. Please replace all `div` and `p` tags with `Box` from MUI